llvm/lib/MC/MCParser/AsmParser.cpp

```diff
@@ -5095,21 +5095,16 @@ bool AsmParser::parseDirectiveComm(bool IsLocal) {
 
 /// parseDirectiveAbort
 ///  ::= .abort [... message ...]
-bool AsmParser::parseDirectiveAbort() {
-  // FIXME: Use loc from directive.
-  SMLoc Loc = getLexer().getLoc();
-
+bool AsmParser::parseDirectiveAbort(SMLoc DirectiveLoc) {
   StringRef Str = parseStringToEndOfStatement();
   if (parseEOL())
     return true;
 
   if (Str.empty())
-    return Error(Loc, ".abort detected. Assembly stopping.");
-  else
-    return Error(Loc, ".abort '" + Str + "' detected. Assembly stopping.");
-  // FIXME: Actually abort assembly here.
```

Comment:
This FIXME hasn't actually been fixed

Reply:
Sorry, accidentally removal :)

Reply:
While here, please drop the trailing period to make the diagnostics more conventional

Reply:
Done

Reply:
The last commit removed the FIXME again

Reply:
Done